src/OSPSuite.Infrastructure.Export/DataRepositoryExportTask.cs

```diff
@@ -108,8 +108,15 @@ private static string tableNameFrom(DataColumn baseGrid)
          var tableName = baseGrid.Repository != null ? baseGrid.Repository.Name : defaultTableName;
          if (string.IsNullOrEmpty(tableName))
             tableName = defaultTableName;
+         var correctedTableName = new string(FileHelper.RemoveIllegalCharactersFrom(tableName).Take(Constants.MAX_NUMBER_OF_CHAR_IN_TABLE_NAME).ToArray());
 
-         return new string(FileHelper.RemoveIllegalCharactersFrom(tableName).Take(Constants.MAX_NUMBER_OF_CHAR_IN_TABLE_NAME).ToArray());
+         while (correctedTableName.EndsWith("'") && correctedTableName.Length > 0)
+            correctedTableName = correctedTableName.Substring(0, correctedTableName.Length - 1);
+
+         while (correctedTableName.StartsWith("'") && correctedTableName.Length > 0)
+            correctedTableName = correctedTableName.Substring(1);
+
+         return correctedTableName;
```

Comment:
weird use case, but if the original name contains only` '`, then correctedTableName will be empty at the end :)

Reply:
yes the point of the test was to make sure that we do not go out ouf bounds in traversing the string.  Empty string is OK: later on the export task replaces the empty names with "Sheet1" etc.

Reply:
there is even this test: When_converting_a_data_repository_with_only_single_quotes for exactly that use case

Reply:
ok, cool